src/modules/dashboard/gameDev/gameDevHandlers/leaderboardGameDevHandlers.ts

```diff
@@ -30,6 +32,19 @@ export const getLeaderboardHandler: Lifecycle.Method = async (request, h) => {
   }
 };
 
+export const getLeaderboardResultsHandler: Lifecycle.Method = async (request, h) => {
+  const leaderboard = await getLeaderboardById(request.params.leaderboardId);
+
+  if (!leaderboard || !request.params.gameTypeId) {
+    throw Boom.notFound('Invalid leaderboard or gametypeId');
+  }
```

Comment:
!request.params.gameTypeId is redundant here since it will already be validated in the middleware

Reply:
Fixed, thanks!